app/controllers/members_controller.rb

```diff
@@ -203,7 +203,7 @@ def create_members
     overall_result
   end
 
-  def with_new_member_params
+  def each_new_member_param
```

Comment:
Renamed because I wanted to make it clearer that this will yield each param and is an iterator